ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+The issues forum is __NOT__ for support requests. It is for bugs and feature requests only.
+Please read https://github.com/angular-ui/bootstrap/blob/master/CONTRIBUTING.md and search
+existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.
```

Comment:
I know you like this, but perhaps just the first line? There is a big warning about reading contributing and the main issue are the support request and you could maintain that.